drivers/i2s/Kconfig.stm32

```diff
@@ -25,3 +39,11 @@ config I2S_STM32_TX_BLOCK_COUNT
 	default 4
 
 endif # I2S_STM32
+
+if I2S_STM32_SAI
+
```

Comment:
I'd propose to add a `I2S_STM32_SAI_DMA` config to enable easy switch between IRQ/DMA

Reply:
done :)